### PR TITLE
prov/gni: Do a noop for setup and teardown for null hints.

### LINKED ISSUE #1281: Add noop teardown function for getinfo_w_null_hints

### PR DESCRIPTION
**Pull request by e-harvey**
gnitest is reporting:  `[----] Warning! The test `rdm_api::getinfo_w_null_hints` crashed during its setup or teardown.`

This PR puts getinfo_w_null_hints in its own test suite.

Fixes #1281.

Signed-off-by: Evan Harvey <eharvey@lanl.gov>

### CODE REVIEW

**Review by sungeunchoi — APPROVED** [review]

**Comment by sungeunchoi** [comment]
How did this get by your testing for #1275?

**Comment by sungeunchoi** [comment]
This needs a signoff.

**Comment by e-harvey** [comment]
I must have gotten lucky when I tested this before merging #1275

**Comment by e-harvey** [comment]
The commit this PR is introducing is signed off. Is there somewhere else I'm supposed to be signing off?

**Comment by e-harvey** [comment]
Oh, never mind. My mistake.

**Pull request merged**
